en/backup-restore-snapshot-perf.md

```diff
@@ -84,26 +84,32 @@ EBS snapshot restore using volumes consists of the following processes. For deta
-| Restores data | 2 to 20 min    | 38%       |  Including the time for restoring data in the raft consensus layer and deleting MVCC data                                 |
-| Starts TiDB   | 1 min        | 2%       |  Including the time for downloading the tidb docker image and starting TiDB                                           |
+| Creates clusters     | 30 seconds         |  2%      | Including the time for downloading docker image and starting PD                                   |
+| Restores volumes     | 20 seconds         |  1%     | Including the time for starting the BR Pod and restoring volumes                                         |
+| Starts TiKV   | 10 to 16 minutes    | 42%      | Including the time for starting RocksDB and reading the meta data of all Regions |
+| Restores data | 2 to 20 minutes    | 52%       |  Including the time for restoring data in the raft consensus layer and deleting MVCC data                                 |
+| Starts TiDB   | 1 minute        | 3%       |  Including the time for downloading the tidb docker image and starting TiDB                                           |
 
 > **Note:**
 >
-> Volume snapshots are consistent upon crash. During restore, before TiKV starts, data self-check and repair are performed, which might take less than 30 minutes. If you restore using high-performance disks that can increase the IOPS and bandwidth, the time can be shortened to 5 minutes. For details about how to use high-performance disks, see [Restore period is excessively long (longer than 2 hours)](backup-restore-faq.md#restore-period-is-excessively-long-longer-than-2-hours).
+> Volume snapshots are consistent upon crash. To start TiKV and restore data during EBS volume snapshot restore, data needs to be initialized first, that is, to be downloaded from Amazon S3. The initialization takes no more than 30 minutes and can be shortened to less than 5 minutes if you use high-performance disks that can increase the IOPS and bandwidth. For details about how to use high-performance disks, see [Restore period is excessively long (longer than 2 hours)](backup-restore-faq.md#restore-period-is-excessively-long-longer-than-2-hours).
 
 ### Restore performance data
 
-The duration of volume snapshot restore depends on the time taken to start TiKV and that for data restore. Time taken to start TiKV might be affected by lazy initialization of EBS volume snapshot restore, for which AWS does not provide quantitative metrics. Test data is as follows under the recommended machine type and GP3 storage volume:
+Time taken by snapshot restore using volumes mainly depends on the time taken by starting TiKV and restoring data. TiKV startup and data restore need to read volume data that is restored from snapshots. Such volume data is loaded with certain latency. Specifically, the data does not reach optimal performance immediately after restore but is available only after it is downloaded from Amazon S3 and written to the volumes.
```

Comment:
The Chinese text is "这是因为，数据只有从 Amazon S3 下载完成...".
```suggestion
Time taken by snapshot restore using volumes mainly depends on the time taken by starting TiKV and restoring data. TiKV startup and data restore need to read volume data that is restored from snapshots. Such volume data is loaded with certain latency. Specifically, the data does not reach optimal performance immediately after restore. This is because the data is available only after it is downloaded from Amazon S3 and written to the volumes.
```